modules/widgets/flickr.php

```diff
@@ -71,8 +71,34 @@ public function widget( $args, $instance ) {
 				? 'https://api.flickr.com/services/feeds/photos_interesting.gne?format=rss_200'
 				: htmlspecialchars_decode( $instance['flickr_rss_url'] );
 
-			// We want to use the HTTPS version so the URLs in the API response are also HTTPS and we avoid mixed-content warnings.
-			$rss_url = preg_replace( '!^http://api.flickr.com/!i', 'https://api.flickr.com/', $rss_url );
+			/**
+			 * Parse the URL, and rebuild a URL that's sure to display images.
+			 * Some Flickr Feeds do not display images by default.
+			 */
+			$flickr_parameters = parse_url( $rss_url );
+			parse_str( $flickr_parameters['query'], $vars );
```

Comment:
@dereksmart Would something like this work for you?
75f18ad19415b1424a5f3d3463d26b20de9328de

We don't accept random URLs anymore.